kratos/solving_strategies/strategies/residualbased_linear_strategy.h

```diff
@@ -610,12 +610,35 @@ class ResidualBasedLinearStrategy
     /**
      * @brief This method directly sets the input as the LHS
      * @param A The LHS matrix
+     * @todo Remove if unused
      */
     void GetDirectSystemMatrix(TSystemMatrixType& A)
```

Comment:
please remove this function. It should not be there! (and i already said this in the review many months back...)